platform/android/java/lib/src/org/godotengine/godot/input/GodotEditText.java

```diff
@@ -155,16 +155,39 @@ public void setView(final GodotRenderView view) {
 	// ===========================================================
 	@Override
 	public boolean onKeyDown(final int keyCode, final KeyEvent keyEvent) {
-		super.onKeyDown(keyCode, keyEvent);
-
-		/* Let GlSurfaceView get focus if back key is input. */
+		/* Let SurfaceView get focus if back key is input. */
 		if (keyCode == KeyEvent.KEYCODE_BACK) {
 			mRenderView.getView().requestFocus();
 		}
 
+		// pass event to godot in special cases
+		if (needHandlingInGodot(keyCode, keyEvent)) {
+			mRenderView.getInputHandler().onKeyDown(keyCode, keyEvent);
+		} else {
+			super.onKeyDown(keyCode, keyEvent);
+		}
```

Comment:
This block needs to be slightly changed to match the pattern pointed out by @m4gr3d:
`if (needHandlingInGodot(...) && inputHandler.onKeyDown(...)) return true else return super.onKeyDown(...)`
to account for the case where the input handler doesn't handle the event.

Reply:
Oh, I missed that, Ill do that later

Reply:
I did that, but for now it's only matter for volume up and down, but that's somthing...